pyproject.toml

```diff
@@ -76,14 +76,14 @@ dev = [
   "build~=1.0",
   "pipdeptree~=2.13",
   "pre-commit~=3.4",
-  "setuptools>=69,<71",
+  "setuptools>=69,<73",
   "twine~=5.1.1",
-  "wheel>=0.42,<0.44",
+  "wheel>=0.42,<0.45",
   "rdmo[allauth]",
   "rdmo[pytest]",
 ]
 gunicorn = [
-  "gunicorn>=21.2,<23.0",
+  "gunicorn>=21.2,<24.0",
```

Comment:
do we need to bump the minimum of the `gunicorn` (https://github.com/benoitc/gunicorn/releases) version here @jochenklar?  
I am out of clue for this

Reply:
yes, go for it (gunicorn is pretty stable I guess, I don't expect problems).